src/style.css

```diff
@@ -1 +1,135 @@
 /* add styles here */
+/* stylelint-disable-next-line block-no-empty */
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf) format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: 700;
+  font-style: normal;
+}
+
+html {
+  margin: 0;
+
+  --main-font: Avenir, Helvetica, Arial, sans-serif;
+  --main-font-weight: 300;
+  --main-font-style: normal;
+  --main-color: rgba(61, 78, 97, 1);
+}
+
+/* search tabs */
+.search__style {
+  display: flex;
+  align-items: center;
+  gap: 17px;
+  margin: 20px;
+
+  box-sizing: border-box;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search__style:hover {
+  box-shadow: 0 2px 2px 1px rgba(61, 78, 97, 0.2);
+  text-shadow: none;
+}
+
+.search__style:focus-within {
+  background: linear-gradient(180deg, #ffff 0%, #f6f6f7 100%);
+}
+
+.search__input-style {
+  outline: none;
+  border: none;
+  box-sizing: border-box;
+
+  font-family: var(--main-font);
+  font-weight: var(--font-weight);
```

Comment:
The CSS variable `--font-weight` is used here, but it is not defined anywhere in the CSS. You might want to define it or use an existing variable like `--main-font-weight`.